maxim/logger/openai/realtime.py

```
@@ -0,0 +1,852 @@
+                f"[MaximSDK][MaximOpenAIRealtimeConnection] Error handling response.created: {str(e)}"
+            )
+
+    def _handle_response_output_audio_delta(
+        self, event: ResponseAudioDeltaEvent
+    ) -> None:
+        """Handle response.output_audio.delta event to accumulate audio chunks."""
+        try:
+            if self._output_audio is None:
+                self._output_audio = bytes()
+            self._output_audio += base64.b64decode(event.delta)
+        except Exception as e:
```

Ruff (0.14.4)

Do not catch blind exception: `Exception`

(BLE001)